db/document.go

```diff
@@ -190,6 +190,24 @@ func (doc *Document) MarshalBodyAndSync() (retBytes []byte, err error) {
 	}
 }
 
+func (doc *Document) MarshalBodyForWebhook() (retBytes []byte, err error) {
+	bodyBytes, err := doc.BodyBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	bodyBytes, err = base.InjectJSONProperties(
```

Comment:
This will be much cheaper if you use `InjectJSONPropertiesFromBytes`, given you only have "simple" types to inject. It requires rolling the JSON-representation of the value yourself (double quotes for strings), but avoids having to marshal each KV pair.

```golang
 	bodyBytes, err = base.InjectJSONPropertiesFromBytes(
		bodyBytes,
		base.KVPairBytes{Key: BodyId, Val: []byte(`"` + doc.ID + `"`)},
		base.KVPairBytes{Key: BodyRev, Val: []byte(`"` + doc.RevID + `"`)},
	)
```